gef.py

```diff
@@ -3246,22 +3246,25 @@ class RemoteCommand(GenericCommand):
     def __init__(self):
         super(RemoteCommand, self).__init__(prefix=False)
         self.add_setting("proc_directory", "/proc")
+        gdb.events.cont.connect(self.continue_handler)
```

Comment:
You are completely right, `new_objfile` is the better solution, I simply overlooked it when reading the docs. I'll fix it today if I have time or else tomorrow.